lib/include/tick/array/sparsearray2d.h

```diff
@@ -89,6 +102,43 @@ class SparseArray2d : public BaseArray2d<T> {
     //! THUS the array becomes a view.
     //! \warning : This method cannot be called on a view
     std::shared_ptr<SSparseArray2d<T>> as_ssparsearray2d_ptr();
+
+    static SparseArray2d<T> FROM_FILE(const std::string& _file){
+      SparseArray2d<T> spd2d;
+      {
+        std::ifstream data(_file, std::ios::in | std::ios::binary);
+        cereal::PortableBinaryInputArchive iarchive(data);
+        iarchive(spd2d);
+      }
+      return spd2d;
+    }
+
+    static void TO_FILE(std::string _file, const SparseArray2d<T>& array){
+      std::ofstream ss(_file, std::ios::out | std::ios::binary);
+      {
+        cereal::PortableBinaryOutputArchive oarchive(ss);
+        oarchive(array);
+      }
+      /** Code to compare serialized versions to original
```

Comment:
I Think this should go in a unit test rather left commented here.

Reply:
A cpp unit test

Reply:
It's a bit more complicated that that as it requires python to be run first but yeah can be

Reply:
We can probably make this run in C++ directly by using this sample matrix
```
   _data  = [ 5 8 3 6 ]
   _row_indices = [ 0 0 2 3 4 ]
   _indices = [ 0 1 2 1 ]
```
(from https://en.wikipedia.org/wiki/Sparse_matrix#Compressed_sparse_row_.28CSR.2C_CRS_or_Yale_format.29)